bucket/legit.json

```diff
@@ -6,7 +6,9 @@
         "identifier": "Freeware",
         "url": "https://github.com/frostming/legit/blob/master/LICENSE"
     },
-    "depends": "git",
+    "suggest": {
+        "git"
```

Comment:
Need to write suggest git like this
https://github.com/ScoopInstaller/Main/blob/3d8c2e1c5eb8adc8fc9627165d0e796cad846876/bucket/git-town.json#L7